x-pack/platform/plugins/private/upgrade_assistant/common/data_stream_types.ts

```diff
@@ -74,27 +74,27 @@ export interface DataStreamReindexStatusNotStarted {
 }
 
 export interface DataStreamReindexStatusInProgress {
-  resolutionType: 'reindex' | 'readonly';
+  resolutionType: 'reindex' | 'readonly' | 'delete';
```

Comment:
Seems like this could be `DataStreamResolutionType` instead of repeating the type content. Could be addressed as a follow up.